components/Form/TextInput/index.js

```diff
@@ -47,7 +47,7 @@ const TextInput = ({
           onChange(e);
         }}
         value={value}
-        aria-describedby={hasError ? `${name}-error` : name}
+        aria-describedby={hasError ? `${name}-error` : `${name}-label`}
```

Comment:
I'm pretty sure this should be as it was at the start ```aria-describedby={hasError ? `${name}-error` : ''}```

Was something complaining about it?

The screen reader will already know that the label describes the element because it has a htmlFor={name} on it

Reply:
If it's complaining that the value is set to nothing, I _think_ setting it to `undefined` will just cause the `aria-describedby` to not be added when there is no error.

Reply:
Yes! Setting it to undefined did the trick, thanks! Totally missed htmlFor ..